packages/core/src/lib/system.ts

```diff
@@ -56,44 +55,26 @@ function getSystemPaths(cwd: string, config: KeystoneConfig) {
   }
 }
 
-function getSudoGraphQLSchema(config: KeystoneConfig) {
-  // This function creates a GraphQLSchema based on a modified version of the provided config.
-  // The modifications are:
-  //  * All list level access control is disabled
-  //  * All field level access control is disabled
-  //  * All graphql.omit configuration is disabled
-  //  * All fields are explicitly made filterable and orderable
-  //
-  // These changes result in a schema without any restrictions on the CRUD
-  // operations that can be run.
-  //
-  // The resulting schema is used as the GraphQL schema when calling `context.sudo()`.
-  const transformedConfig: KeystoneConfig = {
+function getInternalGraphQLSchema(config: KeystoneConfig) {
+  // omit `graphql.omit`
+  const withoutOmit: KeystoneConfig = {
     ...config,
-    ui: {
-      ...config.ui,
-      isAccessAllowed: allowAll,
-    },
     lists: Object.fromEntries(
       Object.entries(config.lists).map(([listKey, list]) => {
         return [
           listKey,
           {
             ...list,
-            access: allowAll,
             graphql: { ...(list.graphql || {}), omit: false },
             fields: Object.fromEntries(
               Object.entries(list.fields).map(([fieldKey, field]) => {
                 if (fieldKey.startsWith('__group')) return [fieldKey, field]
                 return [
                   fieldKey,
-                  (data: FieldData) => {
+                  data => {
                     const f = field(data)
                     return {
                       ...f,
-                      access: allowAll,
-                      isFilterable: true,
-                      isOrderable: true,
```

Comment:
Doesn't isFilterable/isOrderable: false do an omit so don't we need to override this in some way?

Reply:
yes, I have added a temporary warning for now about this